lib/middleware/request-handler.ts

```diff
@@ -191,11 +191,13 @@ export function onRequest(req: Interface.IOurRequest,
         return next(new restify.InternalError('Error not find blpSession.'));
     }
 
+    // TODO validate query params.
```

Comment:
I was gonna add some comments about validating the query parameters before seeing this **TODO** item. :smile: